domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -116,7 +128,12 @@
 
   private static String          configPath               = null;
 
-  private static MultiKeyMap<Object, PostgresClient> connectionPool = MultiKeyMap.multiKeyMap(new HashedMap<>());
+  /**
+   * Used only if {@link #sharedPgPool} is true.
+   */
+  private static final Map<Vertx,PgPool> pgPools = new HashMap<>();
+  /** map (Vertx, String tenantId) to PostgresClient */
```

Comment:
And now, SQ complains.. I wished it would only do that for public static final(s).